src/ape/api/query.py

```diff
@@ -15,21 +15,41 @@
 ]
 
 
-def validate_and_expand_columns(columns: List[str], all_columns: List[str]) -> List[str]:
+def validate_and_expand_columns(columns: List[str], Model: Type[BaseInterfaceModel]) -> List[str]:
     if len(columns) == 1 and columns[0] == "*":
-        return all_columns
+        # NOTE: By default, only pull explicit fields
+        #       (because they are cheap to pull, but properties might not be)
+        return list(Model.__fields__)
 
     else:
-        if len(set(columns)) != len(columns):
+        deduped_columns = set(columns)
+        if len(deduped_columns) != len(columns):
```

Comment:
considering deduped_columns would inherently remove duplicates, should we just fire off a warning letting the user know that they put duplicate fields in and we removed them? No need to raise here if we do so